ReferenceBackendAPI.md

```diff
@@ -0,0 +1,195 @@
+# Reference Backend API
+
+## HTTP Method
+
+The reference backend is deployed to [Twilio Functions](https://www.twilio.com/docs/runtime/functions), a serverless environment.
+
+Twilio Functions does not utilize HTTP methods so you can just use `POST` for all requests.
+
+## Authentication
+
+All requests must set the `Authorization` header to the correct [passcode](README.md#deploy-the-app-to-twilio). The backend will validate the passcode before processing each request.
+
+If you encounter a `401` error, it may be caused by an incorrect passcode. In addition to using the passcode to authenticate each request, the client also derives part of the backend URL from the passcode. 
+
+## Errors
+
+All error responses have this shape:
+
+```json
+{
+  "error": {
+    "message": "error creating room",
+    "explanation": "Room already exists."
+  }
+}
+```
+
+## Endpoints
+
+### /verify-passcode
+
+Verifies that the passcode in the request header is correct. The client can call this endpoint to validate a passcode the user has entered.
+
```

Comment:
It might seem duplicative, but just one sentence here could be nice. Something like:

```
Invalid or expired passcodes will return a `401` error code with an accompanying error object as described above. 
```